### ISSUE TITLE
Fuzzy learning doesn't work in WebUI

### ISSUE THREAD

**Issue by moisseev** [vstakhov/rspamd#289]
After pressing 'Upload text' button I get 'Cannot upload data, no server found' error message.
```
2015-05-30 09:37:50 #76901(controller) rspamd_http_router_finish_handler: requested known path: /auth
2015-05-30 09:37:50 #76901(controller) radix_find_compressed: trying to find key
2015-05-30 09:37:50 #76901(controller) radix_find_compressed: finding value cur value: 4294967295, left: 29247038, right: 00000000, go left
2015-05-30 09:37:50 #76901(controller) radix_find_compressed: finding value cur value: 4294967295, left: 29247024, right: 2924704C, go left
2015-05-30 09:37:59 #76901(controller) rspamd_http_router_finish_handler: requested known path: /learnfuzzy
2015-05-30 09:37:59 #76901(controller) rspamd_controller_error_handler: http error occurred: Not found
2015-05-30 09:37:59 #76901(controller) rspamd_http_router_finish_handler: path: /learnfuzzy not found
2015-05-30 09:38:00 #76901(controller) rspamd_http_router_finish_handler: requested known path: /auth
2015-05-30 09:38:00 #76901(controller) radix_find_compressed: trying to find key
2015-05-30 09:38:00 #76901(controller) radix_find_compressed: finding value cur value: 4294967295, left: 29247038, right: 00000000, go left
2015-05-30 09:38:00 #76901(controller) radix_find_compressed: finding value cur value: 4294967295, left: 29247024, right: 2924704C, go left
```

**Comment by manishmalik** [comment]
Yeah , currently fuzzy learning web interface isn't connected with the backend. We don't have any handler made for it at the worker.

**Comment by vstakhov** [comment]
Pardon? We have a path in the controller named `/fuzzyadd`. You might just check how client send this request.

**Comment by moisseev** [comment]
Fixed with 4844423a36bc20e5f2aac4ebe9107bfb29cb855e
